src/pages/conference/people/staff/detail/userInfo/tab.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { Tabs, Tab } from 'material-ui/Tabs';
 // From https://github.com/oliviertassinari/react-swipeable-views
 import SwipeableViews from 'react-swipeable-views';
-import PersonalInfo from '../personal-info/personalInfo';
+import PersonalInfo from '../personalInfo/personalInfo';
```

Comment:
This line should be `import ... from '../personalInfo'`;
Please use index.js instead.

Reply:
I will create a file for all regrex but there are some problem that i will discuss with you about PersonalInfo tomorow

Reply:
Sure, I am very appreciated for that.